### ISSUE TITLE
Dual UMIs in FASTQ read names are parsed improperly 

### ISSUE THREAD

**Issue by lbeltrame** [bcbio/bcbio-nextgen#3534]
**Version info**
- bcbio version (`bcbio_nextgen.py --version`): master
- OS name and version (`lsb_release -ds`): Debian GNU/Linux 10 (buster)

**Observed behavior**

bcbio will not parse dual UMIs correctly if they're in the read name, in the form `UMI_XXXX-YYYY`. The reason lies in a bug in `umis bamtag` which breaks dual UMIs (only reports the first barcode), fixed by https://github.com/vals/umis/pull/67.  Thus, UMI detection will not use the right fgbio approach because it will mistake them as single UMIs.

This issue is more to keep track and update umis once the relevant PR is merged.

There's another issue with `bcbio_fastq_umi_prep.py`, which I'll file separately.
